elmslie-core/build.gradle

```diff
@@ -11,10 +11,6 @@ dependencies {
     testImplementation(project(":elmslie-test"))
 }
 
-compileKotlin {
-    kotlinOptions.jvmTarget = "1.8"
```

Comment:
I thought that it's not related to AGP but to Kotlin plugin, and default target 1.8 for Kotlin is only available at 1.5.0

Reply:
Thank you for the heads up, i'll update kotlin version in a separate pr. 
I guess it wasn't necessary before, we don't compile kotlin modules for java 8 in our main app as well, so it woudn't be a huge problem here.